docs/payments/concepts/payment-rails.md

```diff
@@ -16,6 +16,62 @@ A Payment Rail is a smart contract-based payment channel that:
 4. **Provides security**: Locks funds to ensure payment obligations are met
 5. **Offers flexibility**: Allows for rate adjustments and termination conditions
 
+### Epoch Duration
+
+In the Filecoin network, one epoch is approximately **30 seconds** in clock time. This is a fundamental parameter in the system because:
+
+1. The `paymentRate` is specified as tokens per epoch
+2. The `lockupPeriod` is measured in epochs (e.g., 2880 epochs = 1 day)
+3. Settlement intervals and proof submission windows are measured in epochs
+
+When calculating costs and rates, this epoch duration must be considered:
+
+```javascript
+// Converting between time-based rates and epoch-based rates
+const dollarsPerMonth = 30.0;
+const secondsPerMonth = 30 * 24 * 60 * 60; // 30 days in seconds
+const secondsPerEpoch = 30; // 30 seconds per epoch
+const epochsPerMonth = secondsPerMonth / secondsPerEpoch; // ~86,400 epochs
+const dollarsPerEpoch = dollarsPerMonth / epochsPerMonth;
+```
+
+### Payment Rate Calculation
+
+The `paymentRate` parameter represents the rate at which funds flow from the client to the storage provider per epoch. This rate is typically calculated off-chain based on several factors:
+
+```javascript
+// Example of calculating payment rate based on data size and market factors
```

Comment:
Give me some. numbers that can make sense so I can see the conversion.  For example, $5/month for 100 GB of data.